frontend/src/app/containers/ExperimentPage.js

```diff
@@ -421,7 +421,7 @@ export class ExperimentDetail extends React.Component {
                   {this.renderLocaleWarning()}
                   {hasAddon && <div>
                     {!!introduction && <section className="introduction">
-                      <div data-l10n-id={this.l10nId('description')} dangerouslySetInnerHTML={createMarkup(introduction)}></div>
+                      <div data-l10n-id={this.l10nId('introduction')} dangerouslySetInnerHTML={createMarkup(introduction)}></div>
```

Comment:
@clouserw so this was a bug. I realize this PR is for content, but this was broken.